website/static/js/notificationsTreebeard.js

```diff
@@ -177,9 +178,18 @@ function ProjectNotifications(data) {
                                     subscribe(item, ev.target.value);
                                 }},
                                 [
-                                    m('option', {value: 'none', selected : item.data.event.notificationType === 'none' ? 'selected': ''}, 'Never'),
-                                    m('option', {value: 'email_transactional', selected : item.data.event.notificationType === 'email_transactional' ? 'selected': ''}, 'Instantly'),
-                                    m('option', {value: 'email_digest', selected : item.data.event.notificationType === 'email_digest' ? 'selected': ''}, 'Daily')
+                                    mentionsInTitle ?
+                                        null :
+                                        m('option', {value: 'none', selected : item.data.event.notificationType === 'none' ? 'selected': ''}, 'Never'),
+                                    mentionsInTitle ?
+                                        null :
+                                        m('option', {value: 'email_transactional', selected : item.data.event.notificationType === 'email_transactional' ? 'selected': ''}, 'Instantly'),
+                                    mentionsInTitle ?
+                                        null :
+                                        m('option', {value: 'email_digest', selected : item.data.event.notificationType === 'email_digest' ? 'selected': ''}, 'Daily'),
+                                    mentionsInTitle ?
+                                        m('option', {value: 'email_transactional', selected: 'email_transactional', disabled: true}, 'Instantly') :
+                                        null,
```

Comment:
[Thinking](https://github.com/leb2dg/osf.io/pull/14#discussion_r65965721) about this more ... we can save ourselves some long lines and I think make this easier to follow if we factor out the array of options for mention notifications vs. other notifications. I'm envisioning something like this just prior to this big ol' `return` statement we're in:

```javascript
if (mentionsInTitle)
    var foo = [m(... 'Instantly')];
else {
    var type = item.data.event.notificationType;
    var foo = [m(...), m(...), m(...)];
}
```